index.md

```diff
@@ -26,6 +26,8 @@ This is a normal paragraph following a header. GitHub is a code hosting platform
 
 ### Header 3
 
+### Header 3
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-duplicate-headings-in-section] Do not use headings with similar content per section (27:1)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)